packages/server/modules/auth/defaultApps.js

```diff
@@ -22,7 +22,7 @@ module.exports = async () => {
   await registerOrUpdateApp({ ...SpeckleConnectorApp })
   await registerOrUpdateApp({ ...SpeckleExcel })
   await registerOrUpdateApp({ ...SpecklePowerBi })
-  await registerOrUpdateApp({ ...SpeckleAutomate })
+  if (speckleAutomateUrl()) await registerOrUpdateApp({ ...SpeckleAutomate })
```

Comment:
Seems like a very localized fix that doesn't resolve the root issue with .env vars overwriting redirectUrls on startup. I'd just have made it so that whenever apps are retrieved on the repo/service level, they're overwritten with whatever's in defaultApps.js (if there are relevant apps there).

Reply:
I tried to do that. It turned out to be a lot more complicated than it should be. When we query the apps back, we add a lot of info around scopes authors etc. That doesn't exist the same way on the default app definitions. It can be done, but its a complex fix for a very specific small problem, that only affects our 'latest' setup.

Reply:
Isn't the same issue gonna happen with other default apps? The web app & API explorer both have redirect URLs dependant on env vars